spec.bs

```diff
@@ -1199,8 +1202,14 @@ A <dfn export>fenced frame config instance</dfn> is a [=struct=] with the follow
 
   : <dfn>embedder shared storage context</dfn>
   :: null, or a [=string=]
-  
+
   : <dfn>is ad component</dfn>
+  :: A [=boolean=]
+
+  : <dfn>can disable untrusted network</dfn>
```

Comment:
copied from [this comment](https://github.com/WICG/fenced-frame/pull/176#discussion_r1748920494):

> Can you add a brief note about how this relates to the disableUntrustedNetwork() API? It seems pretty clear that it controls the behavior of that API in some way, but (a) clearly enunciating that would be great, and (b) describing what conditions make this false would be even better. (I.e., why would a fenced frame not be able to disable its own network?).

Reply:
I don't think this should actually be in the spec. From [the equivalent field in the impl](https://source.chromium.org/chromium/chromium/src/+/main:content/browser/fenced_frame/fenced_frame_config.h;l=664;drc=57c74b90d4a6e506099c97022377d84f523b37cb):

> Currently true in all fenced frame configs, but set to false if loaded in a urn iframe

I think what we want instead is to just check if the relevant traversable navigable is a fenced navigable to see if we're in a fenced frame tree or not. The intention is to make sure that we can't call this from within a urn iframe. I'll remove this field and add the explicit non-fenced frame check in the disableUntrustedNetwork() definition.

Reply:
Does that mean the equivalent field in the implementation can be removed, in favor of the the same check you describe? I'm asking because if the answer is no, then maybe there is some observable difference between what we'd end up spec'ing (without the member) and the implementation (which _does_ use the member, and deviates from the spec in a subtle way).

Reply:
The main difference is that the impl accounts for the existence urn iframes, where it disallows disabling untrusted network across the board. Once urn iframes are removed, the check will strictly be a cross-origin check.